### DISCUSSION TITLE
"skip" option is misleading

### DISCUSSION THREAD

**Discussion by Narretz** [Akryum/vue-apollo#654] [General]
The docs say: 

> Skipping the query

> If the query is skipped, it will disable it and the result will not be updated anymore. You can use the skip option:

To me that means as long as skip is true, the query is not fired. That much is true.

However when "skip" changes from `true` to `false`, the query is executed. This is imo totally unexpected and not the meaning of skip.
Conside the following use case:

I have a query that polls every 10 seconds. However, sometimes I want to skip this query. I set skip to true, and that stops the polling. Later I set skip to false again, and when it's time to poll, then I want the query to fire. Not immediately after I set it to false.
At the moment skip behaves more like enabled / disabled. But skip to me means "a query that was planned should not happen". The skip option itself should never trigger a query.

**Comment by Akryum** [comment]
Since it's expected behavior, maybe the option should be renamed to something like `disable`.

**Comment by Akryum** [comment]
I have added a note about this in the docs in the meantime.
